samples/10.b.customization-password-input/README.md

```diff
@@ -25,308 +25,313 @@ A simple web page with a maximized Web Chat and hosted using React. This sample
 
 We'll start by using the [host with React sample](../03.a.host-with-react) as our Web Chat React template.
 
-First we will build a class-based React component called `PasswordInputActivity`. This component will have a state object with `code` and `submitted` values, set to an empty string and false, respectively.
+In this sample, we will build a new React component that will be show when we receive an event activity from the bot named `passwordInput`. It will show a password box. When the user press <kbd>ENTER</kbd> to submit the box, we will send a post back activity back to the bot.
```

Comment:
```suggestion
In this sample, we will build a new React component that will be shown when we receive an event activity from the bot named `passwordInput`. It will show a password box. When the user presses <kbd>ENTER</kbd> to submit the box, a post back activity will be sent back to the bot.
```